src/Components/Analyzers/src/Resources.resx

```diff
@@ -165,4 +165,13 @@
   <data name="ComponentParameterShouldNotBeSetOutsideOfTheirDeclaredComponent_Title" xml:space="preserve">
     <value>Component parameter should not be set outside of its component.</value>
   </data>
+  <data name="DoNotUseRenderTreeTypes_Description" xml:space="preserve">
+    <value>The types in 'Microsoft.AspNetCore.Components.RenderTree' are not recommended for use outside of the Blazor framework. These  type definitions will change in future major releases.</value>
+  </data>
+  <data name="DoNotUseRenderTreeTypes_Format" xml:space="preserve">
+    <value>The type or member {0} is is not recommended for use outside of the Blazor frameworks. Types defined in 'Microsoft.AspNetCore.Components.RenderTree' will change in future major releases.</value>
```

Comment:
```suggestion
    <value>The type or member {0} is is not recommended for use outside of the Blazor framework. Types defined in 'Microsoft.AspNetCore.Components.RenderTree' will change in future releases.</value>
```

I know we *hope* not to change it outside major releases, but we don't have to give a written promise here :)

Reply:
heh

Reply:
I'm ok with changing the text. I also don't think we'll be allowed to change this in a minor release.